pandas/core/generic.py

```diff
@@ -1016,6 +1016,64 @@ def __setstate__(self, state):
     # ----------------------------------------------------------------------
     # I/O Methods
 
+    _shared_docs['to_excel'] = """
+    Write %(klass)s to a excel sheet
+
+    %(versionadded_to_excel)s
+
```

Comment:
Can you remove the blank lines before and after the versionadded here? And include the newlines in the `versionadded_to_excel` arg for Series. Because otherwise for DataFrame.to_excel, you would get three blank lines instead of one here.